revolv/payments/services.py

```diff
@@ -13,12 +15,14 @@ class PaymentServiceException(Exception):
 class PaymentService(object):
 
     @classmethod
-    def create_payment(cls, user, amount, payment_instrument):
+    def create_payment(cls, user, entrant, amount, project, payment_instrument):
         """
         Create a payment based on a configured payment_instrument.
 
         :user: a User making the payment
+        :entrant: a User entering the payment
```

Comment:
Can we add something about that if the user is the same as the entrant, it represents an organic donation?